.github/workflows/reusable-prober.yml

```diff
@@ -6,6 +6,9 @@ on:
       PAGERDUTY_INTEGRATION_KEY:
         description: 'Integration key for PagerDuty'
         required: true
+      sigstore_github_token:
```

Comment:
To make this usable for private deployments of the prober, could we also add a variable for the repo?

Reply:
The reusable workflow can't use repo variables directly, this secret input is passed in from the caller workflow and is the same as the `SIGSTORE_BOT_PAT` variable. https://docs.github.com/en/actions/using-workflows/reusing-workflows#using-inputs-and-secrets-in-a-reusable-workflow

Reply:
Ah sorry, I meant specifying the repo where the issue would be created as an input variable.

Reply:
Oh gotcha, misunderstood. I can add that.